tests/argumentcount_negative/argumentcount_negative.fz

```diff
@@ -23,38 +23,38 @@
 #
 # -----------------------------------------------------------------------
 
-argumentcount_negative is
+private:public argumentcount_negative is
```

Comment:
Why private:public?

Reply:
I think there was some error that it redefines a feature with more restrictive visibility. But now that I look at it again, I think it might not be necessary?

Reply:
Or just use public

Reply:
seems to no longer be necessary